### PR TITLE
Virtual Keyboard API: link to it from our readme.md

### PR DESCRIPTION
**Pull request by travisleithead**
Quick link redirect.

### CODE REVIEW

**Comment by travisleithead** [comment]
Hmm, maybe that link isn't correct--not sure if Virtual Keyboard Policy has been incorporated with Virtual Keyboard API (which is not listed). Should we have two links? One for the Virtual Keyboard API, and another for the Policy addition?

**Comment by travisleithead** [comment]
Added the VK API, since it's already noted in the charter. Put the link back to the VK Policy.

Note: I can help formerly adopt the VK Policy document over if desired. Filed https://github.com/w3c/editing/issues/322

**Review by johanneswilm — APPROVED** [review]

**Comment by johanneswilm** [comment]
Approved this change, but let's also make sure the Microsoft team that has been working on this agrees.

File: README.md [hunk 1]
```diff
@@ -13,7 +13,8 @@ Details can be found in the [Web Editing Working Group Charter](https://www.w3.o
 * [Clipboard APIs](https://www.w3.org/TR/clipboard-apis/)
 * [Input Events](https://w3c.github.io/input-events/)
 * [Content Editable Disabled](https://w3c.github.io/editing/docs/contentEditableDisabled/)
-* [Virtual Keyboard Policy](https://github.com/MicrosoftEdge/MSEdgeExplainers/blob/master/VirtualKeyboardPolicy/explainer.md)
+* [Virtual Keyboard API](https://w3c.github.io/editing/docs/virtualkeyboard/index.html)
+* [Virtual Keyboard Policy](https://github.com/MicrosoftEdge/MSEdgeExplainers/blob/main/VirtualKeyboardPolicy/explainer.md)
```
Comment (BoCupp-Microsoft) [inline README.md]: We have merged both policy / API explainers for virtual keyboard into one spec called Virtual Keyboard API.  Delete the Virtual Keyboard Policy line; I think the Virtual Keyboard API link above covers it.

**Comment by travisleithead** [comment]
Thanks Bo for the clarification. Merging in now.

**Pull request merged**